ansible/roles/runner/tasks/setup.yml

```diff
@@ -67,3 +67,6 @@
     name: redis
     enabled: true
     state: started
+
+- name: configure size of systemd journal
+  shell: echo "SystemMaxFileSize={{ limit_size_systemd_journal }}" >> /etc/systemd/journald.conf
```

Comment:
This adds the config line every time the playbook is executed. Please use the [lineinfile](http://docs.ansible.com/ansible/latest/lineinfile_module.html) module to avoid this.